arm-dns/2016-04-01/swagger/dns.json

```diff
@@ -937,6 +937,7 @@
         "TTL": {
           "type": "integer",
           "format": "int64",
+	   "x-ms-client-name": "ttl",
```

Comment:
@amarzavery : `x-ms-client-name` is the same as the top level name but lower letters. Based on your talk today afternoon, is this valid? or a no-op?